src/page/CheckRouter.tsx

```diff
@@ -50,6 +52,12 @@ export function CheckRouter() {
         <Route path={`${path}/edit/:id`} exact>
           <CheckEditor onReturn={returnToList} checks={checks} />
         </Route>
+        <Route path={`${path}/type`} exact>
+          <ChooseCheckType onReturn={returnToList} checks={checks} />
+        </Route>
+        <Route path={`${path}/new-multi`} exact>
```

Comment:
Might make sense to use a param variable here. So like `${path}/:checkType`. Then in the form you can grab the `useParams` hook and pull the value of the check type out of the url

Reply:
So to thought if that second part, but the reason I didnt pass the checkType as a query param is because the current value stored in `CheckFormValues['checkType']` looks like this `{label: 'MULTI-HTTP', value: 'multihttp'}`. So the current function `selectableValueFrom` wouldn't work right, and I could write another onemaking an exception for the label `MULTI-HTTP'` because it's CheckType is snake case, not kebab. Anywho, that didnt feel particularly perfect in comparison , since the state exists on `location` so you can pass and access these sorts of things. 

But the first part your comment to adjust the routing altogether to `${path}/:checkType` I'm fine with doing, but I didnt want to be presumptuous. We'll have to adjust all event tracking I think as a result.

Reply:
Yeah maybe let's pair on it for a minute today. I think there might be an easy way to reconcile all that